core/log/metric/aggregator.go

```diff
@@ -119,6 +120,7 @@ func aggregateIntoMap(mm metricTimeMap, metrics map[uint64]*base.MetricItem, nod
 	}
 }
 
+// TODO Might the condition should be stricter here?
 func isActiveMetricItem(item *base.MetricItem) bool {
 	return item.PassQps > 0 || item.BlockQps > 0 || item.CompleteQps > 0 || item.ErrorQps > 0 ||
```

Comment:
Might the condition should be stricter here?
Such as: Resource,Classification,Timestamp.

Reply:
The function is designed for filtering "active" items (already assumed valid), while "active" means it has non-positive metric value.

Reply:
OK~thanks